src/grammar.json

```diff
@@ -738,8 +738,124 @@
     ],
 
     "card": [
-        "Air Elemental",
+        "#cardAlone#",
+        "#cardAlone#",
+        "#cardAlone#",
+        "#cardAlone#",
+        "#cardAlone#",
+        "#cardAlone#",
+        "#cardAlone#",
+        "#cardAlone#",
+        "#descriptiveCard#"
+    ],
+
+    "descriptiveCard": [
+        "an altered #realCard#",
+        "an artist-signed #realCard#",
+        "an HP coffee-stained #realCard#",
+        "a counterfeit #realCard#",
+        "a foil #realCard#",
+        "a foreign-language #realCard#",
+        "a From the Vaults #realCard#",
+        "a proxy #realCard#",
+        "#realCard.a# borrowed from a dealer",
+        "#pro#'s personal #realCard#"
+    ],
+
+    "cardAlone": [
+        "#realCard#",
+        "#realCard#",
+        "#realCard#",
+        "#realCard#",
+        "#realCard#",
+        "#realCard#",
+        "#realCard#",
+        "#realCard#",
+        "#realCard#",
+        "#realCard#",
+        "#fakeCard#"
+    ],
+
+    "realCard": [
```

Comment:
I don't think it's necessary to weight this so heavily toward "normal" cards, since they aren't necessarily funnier. Splitting them evenly is probably fine.